### PR TITLE
MCJ_11_FIX

### PR DESCRIPTION
**Pull request by lenshuygh**
- move GUI settings to CSS file
- created ViewManager to handle FXML loading into 1 window
- removed unused variables

### CODE REVIEW

File: src/be/mcjava/view/CustomerMainMenuOverview.fxml [hunk 1]
```diff
@@ -11,17 +11,21 @@
...
+            <Label text="McMenus"/>
+        </VBox>
         <GridPane fx:id="maingrid">
 
         </GridPane>
-        <VBox fx:id="productsvbox" prefHeight="40"/>
+        <VBox fx:id="productsvbox" prefHeight="40">
+            <Label text="Products"/>
+        </VBox>
         <HBox>
```
Comment (Fonthom) [inline src/be/mcjava/view/CustomerMainMenuOverview.fxml]: I found that you can center these within 1 HBox all the others are redundant.
```
<HBox alignment = "CENTER" spacing = "175">
...(Labels)
<Hbox/>
```

Comment (lenshuygh) [reply src/be/mcjava/view/CustomerMainMenuOverview.fxml]: good tip, these will be replaced by PremadeOrderMenu objects, still implemented your change so i can use it when letting the controller fill this section

**Review by Fonthom — APPROVED** [review]

File: src/be/mcjava/controller/ViewManager.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        stage.setMinWidth(width);
+        stage.setMaxWidth(width);
+        stage.setWidth(width);
+        stage.setMinHeight(height);
+        stage.setMaxHeight(height);
+        stage.setHeight(height);
+    }
+
+    public static void setSceneWidth(double width){
+        stage.setMinWidth(width);
```
Comment (ThomassenCathleia) [inline src/be/mcjava/controller/ViewManager.java:35]: Shouldn't this be scene instead of stage?

Comment (lenshuygh) [reply src/be/mcjava/controller/ViewManager.java:35]: nopez, stage, but i should change the methodName then ...

**Review by ThomassenCathleia — APPROVED** [review]

**Pull request merged**